ingestion-data/production/dataset-config/tornadoes-2024-dow-sw-greenfield.json

```
@@ -0,0 +1,79 @@
+{
+    "collection": "tornadoes-2024-dow-sw-greenfield",
+    "title": "DOW7 Spectrum Width (Greenfield, IA)",
+    "description": "Spectrum Width (m/s) collected by DOW7 from the Greenfield, Iowa EF-4 tornado on May 21, 2024 at 20:41:08.",
```

```suggestion
    "description": "Spectrum Width values (m/s) from the EF-4 Greenfield, IA tornado on May 21, 2024 collected by DOW7. Scan taken at 20:41:08.",
```

just a slight description difference